src/components/Countries/SubComponent/DropdownFilter/DropdownFilter.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
+import Link from 'next/link'
+import Style from './DropdownFilter.module.scss'
+import DropdownSelect from '@/components/common/form-control/DropdownSelect'
+
+const REGION_ITEMS = ['Africa', 'America', 'Asia', 'Europe', 'Oceania']
+export default function DropdownFilter({
+  searchParams,
+}: {
+  searchParams: {
+    region: string
+    search: string
+  }
+}) {
+  const { region, search } = searchParams
+
+  return (
+    <DropdownSelect
```

Comment:
Since you've created a custom select, do you plan to support the native behavior of select to support accessibility? e.g: the up and down arrow